app/containers/TextInput.js

```diff
@@ -26,7 +26,7 @@ const styles = StyleSheet.create({
 		...sharedStyles.textRegular,
 		height: 48,
 		fontSize: 16,
-		paddingHorizontal: 14,
+		padding: 14,
```

Comment:
We're changing from a horizontal padding to a padding horizontal and vertical.
What's the impact of this? Are we touching the vertical padding?